PDF_UA/2/8.2 Logical structure/8.2.5 Additional requirements for specific structure types/8.2.5.2 Document and DocumentFragment/verapdf-profile-8-2-5-2-t02.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<profile xmlns="http://www.verapdf.org/ValidationProfile" flavour="PDFUA_2">
+    <details creator="veraPDF Consortium" created="2023-03-13T10:58:04.137+03:00">
+        <name>ISO 14289-2:202x - 8.2 Logical structure - 8.2.5.2 Document and DocumentFragment - Struct tree root child</name>
+        <description>The structure tree root shall contain a single Document structure element as its only child. 
+            The namespace for that element shall be specified as the PDF 2.0 namespace</description>
+    </details>
+    <hash></hash>
+    <rules>
+        <rule object="PDStructTreeRoot">
+            <id specification="ISO_14289_2" clause="8.2.5.2" testNumber="2"/>
+            <description>The structure tree root shall contain a single Document structure element as its only child. 
+                The namespace for that element shall be specified as the PDF 2.0 namespace</description>
+            <test>kidsStandardTypes != 'Document' || firstChildStandardTypeNamespaceURL == 'http://iso.org/pdf2/ssn'</test>
+            <error>
+                <message>The structure tree root contains a single Document structure element, but that element not 
```

Comment:
The structure tree root contains a single Document structure element, but this element is not within the PDF 2.0 namespace (namespace URI = %1)